plugin/httptrace/httptrace.go

```diff
@@ -36,16 +37,24 @@ var (
 
 // Returns the Attributes, Context Entries, and SpanContext that were encoded by Inject.
 func Extract(ctx context.Context, req *http.Request) ([]core.KeyValue, []core.KeyValue, core.SpanContext) {
-	sc := propagator.Extract(ctx, req.Header)
+	sc, correlationCtx := propagator.Extract(ctx, req.Header)
 
 	attrs := []core.KeyValue{
 		URLKey.String(req.URL.String()),
 		// Etc.
 	}
+	correlationCtx.Foreach(func(kv core.KeyValue) bool {
+		attrs = append(attrs, kv)
+		return true
+	})
```

Comment:
should this be appended to attributes or returned second argument. I believe 'Context Entries' refer to DistributedContext here. @jmacd , can you please confirm?

Reply:
Ah, I suspect you are right, this should be returned as the second arg.

Reply:
(I'm open to improvements on this API.)